library/std/src/process/tests.rs

```diff
@@ -417,6 +417,44 @@ fn env_empty() {
     assert!(p.is_ok());
 }
 
+#[test]
+#[cfg(not(windows))]
+#[cfg_attr(any(target_os = "emscripten", target_env = "sgx"), ignore)]
+fn main() {
+    const PIDFD: &'static str =
+        if cfg!(target_os = "linux") { ", create_pidfd: false" } else { "" };
+
+    let mut command = Command::new("some-boring-name");
+
+    assert_eq!(
+        format!("{command:?}"),
+        format!(
+            r#"Command {{ program: "some-boring-name", args: ["some-boring-name"], env: CommandEnv {{ clear: false, saw_path: false, vars: {{}} }}, cwd: None, uid: None, gid: None, saw_nul: false, groups: None, stdin: None, stdout: None, stderr: None, pgroup: None{PIDFD} }}
+Command line: "some-boring-name""#
+        )
+    );
+
+    command.args(&["1", "2", "3"]);
+
+    assert_eq!(
+        format!("{command:?}"),
+        format!(
+            r#"Command {{ program: "some-boring-name", args: ["some-boring-name", "1", "2", "3"], env: CommandEnv {{ clear: false, saw_path: false, vars: {{}} }}, cwd: None, uid: None, gid: None, saw_nul: false, groups: None, stdin: None, stdout: None, stderr: None, pgroup: None{PIDFD} }}
```

Comment:
So would it be better to show as `inherit` or not show at all when set to `None`?

Reply:
🤔 since we have a lot of properties and most of them can be inherited printing lots of `inherit` wouldn't add much value either. So omitting them seems the best option to me.

Reply:
Hidden them if set to None